modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/js/UserNameFields.es.js

```diff
@@ -191,7 +188,7 @@ class UserNameFields extends PortletBase {
 	 * @protected
 	 */
 	_populateData() {
-		for (const [name, value] of Object.entries(this._formDataCache)) {
+		Object.entries(this._formDataCache).forEach((value,name) => {
```

Comment:
1. This will result in a bug. It should be `forEach(([name, value]) => ... `
1. Thou shall not chain!